app/browser/api/ledger.js

```diff
@@ -2369,6 +2371,7 @@ const initialize = (state, paymentsEnabled) => {
     // First run
     module.exports.checkPublisherInfoUpdate(state)
     // Check again every other day
+    // TODO I think that we need to clear this interval if user turns payments off
```

Comment:
@ryanml

Reply:
Yep, agreed. No purpose for at that point. List fetches will continue if they reenable

Reply:
done